File: AppGenerateFiles/helpdesk/FrontModel/CaseTable_Agenda.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Cat_Dependencias }  from './Cat_Dependencias.js'
import { Tbl_Profile }  from './Tbl_Profile.js'
class CaseTable_Agenda extends EntityClass {
   constructor(props) {
       super(props, 'EntityHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ IdAgenda;
   /**@type {String}*/ Dia;
   /**@type {String}*/ Hora_Inicial;
   /**@type {String}*/ Hora_Final;
   /**@type {Date}*/ Fecha_Caducidad;
   /**@type {Cat_Dependencias} ManyToOne*/ Cat_Dependencias;
   /**@type {Tbl_Profile} ManyToOne*/ Tbl_Profile;
}
export { CaseTable_Agenda }

File: AppGenerateFiles/helpdesk/Model/CaseTable_Agenda.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class CaseTable_Agenda : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? IdAgenda { get; set; }
       public int? Id_Perfil { get; set; }
       public int? Id_Dependencia { get; set; }
       public string? Dia { get; set; }
       public string? Hora_Inicial { get; set; }
       public string? Hora_Final { get; set; }
       public DateTime? Fecha_Caducidad { get; set; }
       [ManyToOne(TableName = "Cat_Dependencias", KeyColumn = "Id_Dependencia", ForeignKeyColumn = "Id_Dependencia")]
       public Cat_Dependencias? Cat_Dependencias { get; set; }
       [ManyToOne(TableName = "Tbl_Profile", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public Tbl_Profile? Tbl_Profile { get; set; }
   }
}
